lms/envs/aws.py

```diff
@@ -586,6 +586,7 @@
 # BAYT KEYS
 BAYT_SECRET_KEY = AUTH_TOKENS.get("BAYT_SECRET_KEY")
 BAYT_BASE = ENV_TOKENS.get("BAYT_BASE")
+BAYT_VALID_YEARS = ENV_TOKENS.get("BAYT_VALID_YEARS")
```

Comment:
Did you know that you can add a default value?

      BAYT_VALID_YEARS = ENV_TOKENS.get("BAYT_VALID_YEARS", 10)

You don't have to do this, though!